code/modules/fishing/fish/_fish.dm

```diff
@@ -3,6 +3,9 @@
 
 #define GET_FISH_ELECTROGENESIS(fish) (fish.electrogenesis_power * fish.size * 0.1)
 
+#define FISH_SUBMERGING_THRESHOLD 80
```

Comment:
Why are we working in ticks here instead of seconds? It works but it seems more confusing than it needs to be.

Reply:
Sure, will change it to seconds when I'm home.